internal/testdata/snapshots/output/inlinestruct/inlinestruct_multivar.go

```diff
@@ -16,7 +16,7 @@
 //           ^^^^^^^^^^ definition 0.1.test `sg/inlinestruct`/emptyMocks.
 //           documentation ```go
    Code func(p Params) (response *HighlightedCode, aborted bool, err error)
-// ^^^^ definition 0.1.test `sg/inlinestruct`/inline-30:Code.
```

Comment:
Test failure...

```
         --- /home/runner/work/scip-go/scip-go/internal/testdata/snapshots/output/inlinestruct/inlinestruct_multivar.go (obtained)
        +++ /home/runner/work/scip-go/scip-go/internal/testdata/snapshots/output/inlinestruct/inlinestruct_multivar.go (expected)
        @@ -16,7 +16,7 @@
         //           ^^^^^^^^^^ definition 0.1.test `sg/inlinestruct`/emptyMocks.
         //           documentation ```go
            Code func(p Params) (response *HighlightedCode, aborted bool, err error)
        -// ^^^^ definition 0.1.test `sg/inlinestruct`/inline-27:Code.
        +// ^^^^ definition 0.1.test `sg/inlinestruct`/inline-18:Code.
```

Is this integer OS-specific?

Reply:
There is an `inlineCount` that was being tracked as a global, and that ends up being OS-specific for some reason. I've replaced that with a line + column combination in the source.